packages/cli/src/commands/apps/oauth/generate.ts

```diff
@@ -24,6 +23,8 @@ export default class AppOauthGenerateCommand extends InputOutputAPICommand<AppOA
 		const { args, argv, flags } = this.parse(AppOauthGenerateCommand)
 		await super.setup(args, argv, flags)
 
-		this.processNormally((data) => { return this.client.apps.regenerateOauth(args.id, data) })
+		this.processNormally(args.id,
+			() => { return this.client.apps.list() },
```

Comment:
When an arrow function has only one command and that command is returned, you can leave off the `return` and the curly brackets. i.e. this line could just be `() => this.client.apps.list(),`